CondFormats/GeometryObjects/interface/PHGCalParameters.h

```diff
@@ -47,6 +56,17 @@ class PHGCalParameters {
   std::vector<double> cellCoarseX_;
   std::vector<double> cellCoarseY_;
   std::vector<double> boundR_;
+  std::vector<double> rLimit_;
+  std::vector<double> waferThickness_;
+  std::vector<double> cellThickness_;
+  std::vector<double> radius100to200_;
+  std::vector<double> radius200to300_;
+  std::vector<double> radiusMixBoundary_;
+  std::vector<double> rMinLayerBH_;
+  std::vector<double> radiusLayer_[2];
```

Comment:
(Just to understand) is this a C-type two dimensional array of std::vector<double>'s?

Reply:
Yes it is